gio/src/lib.rs

```diff
@@ -7,6 +7,7 @@
 #![allow(clippy::missing_safety_doc)]
 #![allow(clippy::upper_case_acronyms)]
 #![allow(clippy::non_send_fields_in_send_ty)]
+#![allow(clippy::should_implement_trait)]
```

Comment:
This is generally a good lint. Can you only allow it for the one instance where it's wrong?

Reply:
That would require another gir update but I can do that.

Reply:
As discussed with @bilelmoussaoui, just put it on the `auto` module for now via `src/lib.rs`